src/main/java/stringcaculator/domain/Numbers.java

```diff
@@ -0,0 +1,19 @@
+package stringcaculator.domain;
+
+import java.util.List;
+
+public class Numbers {
+
+    private final List<Number> numbers;
+
+    public Numbers(List<Number> numbers) {
+        this.numbers = numbers;
+    }
+
+    public int sum() {
+        return numbers.stream()
+            .map(Number::value)
+            .mapToInt(i -> i)
+            .sum();
```

Comment:
`map`과 `mapToInt`를 호출할 필요없이 한번으로 끝낼 수 있을 것 같아요!

```suggestion
        return numbers.stream()
            .mapToInt(Number::value)
            .sum();
```